.exp-insertions.yml

```diff
@@ -38,12 +38,12 @@ pool:
   vmImage: windows-latest
 
 resources:
-  repositories: 
+  repositories:
```

Comment:
Minor: This resource is not needed anymore, let's delete it.

Reply:
Done